### PR TITLE
resource_control: add some metrics for background task control

### PR DESCRIPTION
**Pull request by glorv**
### What is changed and how it works?

Issue Number: ref #14900 

What's Changed:

```
Add some metrics for background task control
```

The corresponding PR for grafana panel is pingcap/tidb#45594
Screen snapshot:

image
image

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by Connor1996 — APPROVED** [review]
LGTM

**Review by HuSharp — APPROVED** [review]
LGTM!

File: metrics/grafana/tikv_details.json [hunk 1]
```diff
@@ -47316,6 +47316,576 @@
...
+          "renderer": "flot",
+          "repeatDirection": "h",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "exemplar": true,
+              "expr": "tikv_resource_control_background_quota_limiter{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\", type=\"cpu\"}",
```
Comment (nolouch) [inline metrics/grafana/tikv_details.json:47389]: How about moving to the independent resource control panel?

Comment (glorv) [reply metrics/grafana/tikv_details.json:47389]: The `instance` used in `resource control` is pd but here the instance is tikv. Not sure if it's a good idea to support multiple `instance` types in the same page

**Comment by glorv** [comment]
@nolouch  PTAL again

**Review by nolouch — APPROVED** [review]
lgtm

**Pull request merged**